app/lib/efile/nj/nj1040.rb

```diff
@@ -95,6 +100,26 @@ def calculate_line_15
         sum.round
       end
 
+      def calculate_line_27
+        calculate_line_15
+      end
+
+      def calculate_line_29
+        calculate_line_27
```

Comment:
[dust] It feels a little labyrinthian to see line 29 refer to line 27 refer to line 15. I wonder if this is a justifiable place to include a comment? Or maybe one func calculate_lines_15_27_29?

Reply:
hm, the thing is that they won't be like that in the future though